WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -612,18 +612,6 @@ final class EditableOrderViewModel: ObservableObject {
         autodismissableNotice = Notice(title: Localization.newTaxRateSetSuccessMessage)
     }
 
-    /// Erases stored tax rate from order by cleaning the address, and removes stored tax rate from storage
-    ///
-    func forgetTaxRate() {
-        let order = orderSynchronizer.order
-        orderSynchronizer.setAddresses.send(OrderSyncAddressesInput(billing: order.billingAddress?.resettingTaxRateComponents(),
-                                                                    shipping: order.shippingAddress?.resettingTaxRateComponents()))
-        resetAddressForm()
-        storedTaxRate = nil
-        stores.dispatch(AppSettingsAction.setSelectedTaxRateID(id: nil, siteID: siteID))
-        autodismissableNotice = Notice(title: Localization.stopAddingTaxRateAutomaticallySuccessMessage)
-    }
-
```

Comment:
Is this correct? It doesn't seem related to the analytics, I'm assuming you're just tidying up something unused...?

Reply:
Yeah, that was a small refactor to make that function private and encapsulate it on `onClearAddressFromBottomSheetTapped` together with the event tracking.